packages/coordinator/tests/services/verification/signature-verification.spec.ts

```diff
@@ -96,12 +96,42 @@ describe("verifyPrescriptionSignature", () => {
 
   describe("Signature is invalid", () => {
     const validSignature = TestResources.parentPrescriptions.validSignature.ParentPrescription
+    const valid256Signature = TestResources.parentPrescriptions.sha256Signature.ParentPrescription
 
     test("passes if prescription has valid Signature that matches prescription", async () => {
       const result = await verifyPrescriptionSignature(validSignature, logger)
       expect(result).not.toContain("Signature is invalid")
     })
 
+    test("passes if prescription signature method algorithm that references SHA-256 matches prescription", async () => {
+      const result = await verifyPrescriptionSignature(valid256Signature, logger)
+      expect(result).not.toContain("Signature is invalid")
+    })
+
+    test("passes if prescription signature method algorithm that references not SHA-256 or SHA-1 matches prescription",
```

Comment:
Clarify the name of this test?
Maybe "passes if prescription signature is valid but method algorithm does not reference SHA-256 or SHA-1"
Or I could just be misunderstanding here, of course.

Reply:
The proposed change has been applied.